File: AFMobile/AFAndroid/app/src/main/java/cz/cvut/fel/matyapav/afandroid/enums/LabelPosition.java
package cz.cvut.fel.matyapav.afandroid.enums;

/**
 * Enum for supported positions of label in form
 *
 * @author Pavel Matyáš (matyapav@fel.cvut.cz).
 * @since 1.0.0..
 */
public enum LabelPosition {

    BEFORE("BEFORE"),AFTER("AFTER"), NONE("NONE");

    private String position;

    LabelPosition(String position) {
        this.position = position;
    }

    public String getPosition() {
        return position;
    }
}

File: AFMobile/AFWinPhone/AFWinPhone/enums/LabelPosition.cs
using System;

namespace AFWinPhone.enums
{
    public sealed class LabelPosition
    {

        public static readonly LabelPosition BEFORE = new LabelPosition("BEFORE");
        public static readonly LabelPosition AFTER = new LabelPosition("AFTER");
        public static readonly LabelPosition NONE = new LabelPosition("NONE");

        private readonly String position;

        LabelPosition(String position)
        {
            this.position = position;
        }

        public String getPosition()
        {
            return position;
        }

    }
}
